hedera-node/src/main/java/com/hedera/services/ledger/TransferLogic.java

```diff
@@ -74,6 +77,8 @@ public class TransferLogic {
     private final TransactionalLedger<
                     Pair<AccountID, TokenID>, TokenRelProperty, MerkleTokenRelStatus>
             tokenRelsLedger;
+    private final TransactionContext txnCtx;
+    private final Supplier<StateView> currentView;
```

Comment:
Since this is only used to resolve aliases, I think `AliasManager` is a better dependency to inject.

Reply:
Fixed !